apps/web/src/components/poll/desktop-poll.tsx

```
@@ -365,13 +365,33 @@ const DesktopPoll: React.FunctionComponent = () => {
-                        variant="primary"
-                        form="voting-form"
-                      >
-                        <Trans i18nKey="continue" />
-                      </Button>
+                      <Tooltip>
+                        <TooltipTrigger asChild>
+                          <Button
+                            type="submit"
+                            variant="primary"
+                            form="voting-form"
+                            className={!votingForm.hasValidDate ? "opacity-50 cursor-not-allowed" : ""}
```

Biome (2.1.2)

[error] These CSS classes should be sorted.

Safe fix: Sort the classes.


(lint/nursery/useSortedClasses)